playbooks/install_os.yml

```diff
@@ -17,10 +32,9 @@
         tasks_from: install.yml
       tags:
         - edpm_sshd
-    - name: Install edpm_chrony
+    - name: Install and configure edpm_chrony
       ansible.builtin.import_role:
-        name: osp.edpm.edpm_chrony
-        tasks_from: install.yml
+        name: fedora.linux_system_roles.timesync
```

Comment:
Do you know if the downstream FCQN will be the same? 

I have no idea, I just didn't want to have to also have a version of this with something like `rhel.linux_system_roles.timesync`. 

Also, what installs this module usptream and downstream?

Reply:
nope, they are a bit different. I already faced this problem calling `network` role in `edpm_network_config`. I build the collection FQCN based on distro -> https://github.com/openstack-k8s-operators/edpm-ansible/blob/26fe00b33b84a05ee7107a704d9dcd2bc1692612/roles/edpm_network_config/tasks/main.yml#L22-L26

Reply:
OK, I guess this patch should borrow from @rebtoor's example

Reply:
> Also, what installs this module usptream and downstream?

https://access.redhat.com/articles/3050101

TL;DR

`dnf -y install rhel-system-roles`

Reply:
In upstream it's ansible galaxy, in downstream it depends. Ansible doesn't support downstream collection releases, or any other delivery mechanism than galaxy. Unless we are building package somewhere, this is the way it will be in downstream.

Technically it's not such a problem, we are sending this over as an image with dependencies installed.

Reply:
I just wanted to know who that image will be built for either case. Thanks to @rebtoor for the answer